src/sources/aws_s3/sqs.rs

```diff
@@ -457,7 +470,7 @@ impl IngestorProcess {
                 let aws_region = Bytes::from(s3_event.aws_region.as_str().as_bytes().to_vec());
 
                 let mut stream = lines.filter_map(move |line| {
-                    emit!(&SqsS3EventReceived {
+                    emit!(&SqsS3EventsReceived {
```

Comment:
Just curious (since this is existing behavior), why do we emit `EventsReceived` here versus after we construct the log event (lines `464-476`)? It seems like technically the `event_bytes` is more accurately represented afterwards.

Reply:
Good point, it was there initially but it definitely belongs afterward.